.github/workflows/static.yml

```diff
@@ -31,13 +30,22 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
```

Comment:
I think that tests need to be run as well 

the CI/CD whole flow:
`-> build -> run the tests -> PR(code-review) -> approval -> merge into master -> deploy to github pages`